### PR TITLE
Archive

### LINKED ISSUE #69: Archive it
As the demo is available for longer than 2 years, we archive it. Todo:

- [x] Create a README.md file (@ineiti)
- [x] Add a Dockerfile to create a self-contained docker-image
- [x] Push the docker-image to ghcr.io (or whatever) 
- [x] Add a docker-compose.yaml for easy startup and update README.md

### PR DESCRIPTION
**Pull request by ineiti**
Making a nice archive with a simple 

```bash
docker-compose up
```

Demo.

Reviews: whoever comes first

Closes #69

### CODE REVIEW

**Review by PascalinDe — COMMENTED** [review]
[.travis.yml](https://github.com/c4dt/service-spindle/blob/main/.travis.yml) is not used anymore, maybe you can delete it on the occasion

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/.git
+/datasets
+/configs
+node_modules/
+dist/
```
Comment (PascalinDe) [inline .dockerignore:6]: maybe add .github as well

File: README.md [hunk 2]
```diff
@@ -1,4 +1,21 @@
...
+This repo has been archived is no longer supported.
+You can however use this as a demo to run on your local machine.
+To run the demo, download the repo, run the `docker-compose.yaml`,
+and point the browser to [localhost:8080](http://localhost:8080):
+
+```bash
+git clone https://github.com/c4dt/service-spindle
+cd service-spindle
+docker-compose up -d
+# At least on Mac you can quickly open the frontend:
```
Comment (PascalinDe) [inline README.md]: personally I find that comment a bit confusing

File: datasets/minmax.sh [hunk 3]
```diff
@@ -0,0 +1,35 @@
+#!/bin/zsh
+
+# This is a very handwavy script to allow to convert the user input values as coming
+# from the measurements to the values used in the nodes to train.
```
Comment (PascalinDe) [inline datasets/minmax.sh:4]: maybe just add a line saying that it generated the data used in [webapp/src/app/query-runner/query-runner.component.ts](https://github.com/c4dt/service-spindle/pull/76/files#diff-22ee2e3457ded6cdd7bd44de8dd8d2037fc77c9a159c84b9ac0690d83aff62d4)

File: webapp/src/app/query-runner/query-runner.component.ts [hunk 4]
```diff
@@ -148,8 +173,10 @@ export class QueryRunnerComponent implements OnChanges {
         }
       },
       (error) => {
-        this.state = ["train error", error];
-        throw error;
+        // Archiving: no idea why this doesn't work anymore. But as the
+        // necessary data got transmitted, it should be OK.
+//         this.state = ["train error", error];
```
Comment (PascalinDe) [inline webapp/src/app/query-runner/query-runner.component.ts]: can you remove the commented duplicate line, it's a bit confusing like this

File: README.md (hunk 2 above)
Comment (ineiti) [reply README.md]: Removed comment.

File: webapp/src/app/query-runner/query-runner.component.ts (hunk 4 above)
Comment (ineiti) [reply webapp/src/app/query-runner/query-runner.component.ts]: removed the `(error)`-callback.

**Comment by PascalinDe** [comment]
LGTM

**Review by PascalinDe — APPROVED** [review]

**Pull request merged**